### PR TITLE
feat: add wwt to api, add sdk docs

### PR DESCRIPTION
**Pull request by DiRaiks**
### Description

- Add WWT to API page
- Add Documentation link for SDK

### Demo

image
image

### Code review notes

Please review WWT description

### CODE REVIEW

File: docs/integrations/sdk.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ React components for Lido Finance projects.
 
 Library for interaction with Lido protocol.
```
Comment (TheDZhon) [inline docs/integrations/sdk.md:19]:
```suggestion
Library for interaction with the Lido on Ethereum protocol.
```

File: docs/integrations/api.md [hunk 2]
```diff
@@ -20,13 +20,13 @@ https://eth-api.lido.fi/v1/protocol/steth/apr/sma
 
 Response schema and examples are available in the [Swagger API documentation](https://eth-api.lido.fi/api/static/index.html#/APR%20for%20Eth%20and%20stEth/ProtocolController_findSmaAPRforSTETH)
```
Comment (loga4) [inline docs/integrations/api.md:21]: seems like the link is broken

File: docs/integrations/api.md [hunk 3]
```diff
@@ -107,3 +107,40 @@ http://reward-history-backend-holesky.testnet.fi/?address=0x12345
 ```
 
 Response schema and examples are available in the [Swagger API documentation](https://reward-history-backend-holesky.testnet.fi/api)
+
+## Withdrawals API
+
+The Withdrawals API service offers an utility for estimating the waiting time for withdrawals within the Lido protocol on Ethereum.
+The service is helpful for stakers, providing insights from the moment of withdrawal request placement to its finalization when the request becomes claimable.
+
+See the [detailed explanation.](https://github.com/lidofinance/withdrawals-api/blob/develop/how-estimation-works.md)
```
Comment (TheDZhon) [inline docs/integrations/api.md:116]:
```suggestion
See the [detailed explanation](https://github.com/lidofinance/withdrawals-api/blob/develop/how-estimation-works.md).
```

File: docs/integrations/api.md [hunk 4]
```diff
@@ -107,3 +107,40 @@ http://reward-history-backend-holesky.testnet.fi/?address=0x12345
 ```
 
 Response schema and examples are available in the [Swagger API documentation](https://reward-history-backend-holesky.testnet.fi/api)
+
+## Withdrawals API
+
+The Withdrawals API service offers an utility for estimating the waiting time for withdrawals within the Lido protocol on Ethereum.
```
Comment (TheDZhon) [inline docs/integrations/api.md:113]:
```suggestion
The Withdrawals API service offers an utility for estimating the waiting time for [withdrawals](https://docs.lido.fi/contracts/withdrawal-queue-erc721) within the Lido on Ethereum protocol.
```

**Review by TheDZhon — APPROVED** [review]
LGTM except only for a few minor suggestions

**Review by loga4 — COMMENTED** [review]
my 2cents here - old swagger links are invalid

File: docs/integrations/api.md (hunk 2 above)
Comment (DiRaiks) [reply docs/integrations/api.md]: Thank you! Fixed.
The Swagger URL was updated this morning.

**Pull request merged**